content/docs/dvclive/api-reference/live/set_step.md

```diff
@@ -1,7 +1,5 @@
 # dvclive.set_step()
 
-Signals that the current step has ended and sets `step` to the given value.
```

Comment:
Same as above. Why get rid of this?

Reply:
That's odd I thought I've added back all short descriptions